src/Data/Impact.elm

```diff
@@ -231,11 +218,9 @@ encodeBonusesImpacts bonuses =
         ]
 
 
-encodeImpacts : Definitions -> Scope -> Impacts -> Encode.Value
-encodeImpacts definitions scope (Impacts impacts) =
+encodeImpacts : Impacts -> Encode.Value
+encodeImpacts (Impacts impacts) =
     Definition.encodeBase
-        definitions
-        scope
         Unit.encodeImpact
         impacts
```

Comment:
Do we really need this proxy, or should we now rather use `Definition.encodeBase` directly? 🤔